src/app/public/modules/checkbox/checkbox.component.spec.ts

```diff
@@ -72,6 +75,26 @@ class CheckboxWithFormDirectivesComponent {
   public isGood: boolean = false;
 }
 
+/** Simple component for testing an MdCheckbox with ngModel. */
+@Component({
+  template: `
+  <div>
+    <form [formGroup]="checkboxForm">
+      <sky-checkbox name="cb" formControlName="checkbox1" #wut>
+        <sky-checkbox-label>
+          Be good
+        </sky-checkbox-label>
+      </sky-checkbox>
+    </form>
+  </div>
+  `
+})
+class CheckboxWithReactiveFormComponent {
```

Comment:
This reflects the current fixtures pattern set in the checkbox spec, but we'll definitely need to create a fixtures module for all of these components in the upcoming "code cleanup" task.

Reply:
Sure :)